packages/atomic/src/components/insight/atomic-insight-interface/atomic-insight-interface.new.stories.tsx

```diff
@@ -0,0 +1,851 @@
+
+export const Default: Story = {
+  name: 'atomic-insight-panel',
+  decorators: [
+    () =>
+      html`<atomic-insight-layout>
+        <style>
+          atomic-insight-search-box {
+            position: relative;
+          }
+
+          atomic-insight-search-box::part(wrapper) {
+            top: 0;
+            left: 0;
+            max-width: unset;
+          }
+
+          atomic-insight-pager {
+            height: 4em;
+          }
+
+          atomic-insight-interface:not([widget='false']),
+          atomic-insight-layout:not([widget='false']) {
+            height: auto;
+          }
+        </style>
+        <atomic-layout-section section="search">
+          <atomic-insight-search-box></atomic-insight-search-box>
+          <atomic-insight-refine-toggle></atomic-insight-refine-toggle>
+          <atomic-insight-user-actions-toggle></atomic-insight-user-actions-toggle>
+          <atomic-insight-tabs>
+            <atomic-insight-tab
+              label="All"
+              expression=""
+              active="true"
+            ></atomic-insight-tab
+            ><atomic-insight-tab
+              label="Videos"
+              expression="@ytchanneltitle"
+            ></atomic-insight-tab
+            ><atomic-insight-tab
+              label="Documentation"
+              expression='@documenttype==("WebPage")'
+            ></atomic-insight-tab>
+          </atomic-insight-tabs>
+        </atomic-layout-section>
+        <atomic-layout-section section="facets">
+          <atomic-insight-timeframe-facet
+            field="date"
+            label="Date"
+            with-date-picker="true"
+          >
+            <atomic-timeframe unit="hour"></atomic-timeframe>
+            <atomic-timeframe unit="day"></atomic-timeframe>
+            <atomic-timeframe unit="week"></atomic-timeframe>
+            <atomic-timeframe unit="month"></atomic-timeframe>
+            <atomic-timeframe unit="quarter"></atomic-timeframe>
+            <atomic-timeframe
+              unit="year"
+            ></atomic-timeframe> </atomic-insight-timeframe-facet
+          ><atomic-insight-facet
+            field="source"
+            label="Source"
+            display-values-as="checkbox"
+          ></atomic-insight-facet
+          ><atomic-insight-facet
+            field="documenttype"
+            label="Item type"
+            display-values-as="checkbox"
+          ></atomic-insight-facet>
+        </atomic-layout-section>
+        <atomic-layout-section section="status">
+          <atomic-insight-query-summary></atomic-insight-query-summary>
+        </atomic-layout-section>
+        <atomic-layout-section section="results">
+          <atomic-insight-generated-answer collapsible="true">
+          </atomic-insight-generated-answer>
+          <atomic-insight-folded-result-list image-size="none">
+            <atomic-insight-result-template must-match-sourcetype="YouTube">
+              <template>
+                <style>
```

Comment:
si tu enleve tout le styling est-ce que c'est laid? 😆

Reply:
nah honestly doesn't look like it changes anything haha